airbyte-integrations/connectors/source-orbit/source_orbit/streams.py

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from abc import ABC
+from typing import Any, Iterable, Mapping, MutableMapping, Optional, cast
+
+import requests
+import urllib.parse
+
+from airbyte_cdk.sources.streams.http import HttpStream
+
+
+class OrbitStream(HttpStream, ABC):
+    url_base = "https://app.orbit.love/api/v1/"
+
+    def __init__(self, workspace: str, **kwargs):
+        super().__init__(**kwargs)
+        self.workspace = workspace
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        return None
+
+    def request_params(
+        self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> MutableMapping[str, Any]:
+        return {}
```

Comment:
```suggestion
```
This function is redundant, as it is already implemented in the HttpStream parent class.